chain/chain/src/store.rs

```diff
@@ -2205,14 +2389,16 @@ mod tests {
     use near_primitives::block::Block;
     use near_primitives::errors::InvalidTxError;
     use near_primitives::hash::hash;
-    use near_primitives::types::{BlockHeight, EpochId};
+    use near_primitives::types::{BlockHeight, EpochId, StateRoot};
     use near_primitives::utils::index_to_bytes;
     use near_primitives::validator_signer::{InMemoryValidatorSigner, ValidatorSigner};
     use near_store::test_utils::create_test_store;
+    use near_store::{Trie, WrappedTrieChanges};
 
     use crate::store::ChainStoreAccess;
     use crate::test_utils::KeyValueRuntime;
-    use crate::{Chain, ChainGenesis, DoomslugThresholdMode};
+    use crate::{Chain, ChainGenesis, DoomslugThresholdMode, Tip};
+    use near_store::trie::gen_changes;
```

Comment:
nit: move the use into the relevant group.

Reply:
Not actual anymore.